File: src/main/java/conditionalStatements/TestScore.java
package conditionalStatements;

import java.util.Scanner;

public class TestScore {

    public static void main(String[] args) {
        // 1. Scanner
        Scanner scanner = new Scanner(System.in);
        int score = scanner.nextInt();

        // 2. BufferedReader
        // BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
        // int score2 = Integer.parseInt(bufferedReader.readLine());

        if (score >= 90) {
            System.out.println("A");
        } else if (score >= 80) {
            System.out.println("B");
        } else if (score >= 70) {
            System.out.println("C");
        } else if (score >= 60) {
            System.out.println("D");
        } else {
            System.out.println("E");
        }
    }

}

File: src/main/kotlin/conditionalStatements/TestScore.kt
package conditionalStatements

import java.util.Scanner

fun main() {
    val scanner = Scanner(System.`in`)
    val score = scanner.nextInt()
    scanner.close()

    when {
        score >= 90 -> println("A")
        score >= 80 -> println("B")
        score >= 70 -> println("C")
        score >= 60 -> println("D")
        else -> println("F")
    }

}